external-apis/muoversi2015.e015.servizirl.it.yaml

```diff
@@ -30,18 +30,65 @@ info:
     name: Technical Management Board di E015
     url: http://www.e015.regione.lombardia.it/PE015/esplora-i-contenuti/i-servizi/muoversi-lombardia
     email: e015-tmb@regione.lombardia.it
+    x-publisher: [dati publisher]
+    x-provider: [dati provider servizi]
+    x-complexity-index: [valore della complessità delle api (in FP)]
+  x-gdpr:
```

Comment:
Questo avrebbe senso gestirlo con la granularità negli schemas, in modo da poter taggare i singoli campi e appoggiarsi ad eventuali schemi metadatabili (Eg. vedi il riferimento che questo schema ha verso le ontologie https://github.com/teamdigitale/openapi/blob/master/docs/schemas/tax_code.yaml)

In questo caso si potrebbe fare anche un lavoro più articolato, collegando i singoli schemi e/o dati alle ontologie agid.

Reply:
totalmente d'accordo, va spostato negli schemas. Il parser lato server può comporre tutte le informazioni relative alla privacy scorrendo le informazioni sulla privacy dei dati che vanno trattati direttamente nella definizione dei dati stessi.

Reply:
Esatto, e le indicazioni potrebbero essere addirittura prese dall'ontologia agid.